voyager/plugins/client-update/mpt-trusted/Cargo.toml

```diff
@@ -0,0 +1,32 @@
+[package]
+name    = "voyager-client-update-plugin-mpt-trusted"
+version = "0.0.0"
+
+authors      = { workspace = true }
+edition      = { workspace = true }
+license-file = { workspace = true }
+publish      = { workspace = true }
+repository   = { workspace = true }
+
+[lints]
+workspace = true
+
+[dependencies]
+alloy                          = { workspace = true, features = ["rpc", "rpc-types", "transports", "transport-http", "transport-ws", "reqwest", "provider-ws"] }
+beacon-api                     = { workspace = true }
+beacon-api-types               = { workspace = true, features = ["serde"] }
+bitvec                         = { workspace = true }
+embed-commit                   = { workspace = true }
+enumorph                       = { workspace = true }
+ethereum-light-client-types    = { workspace = true, features = ["serde"] }
+ethereum-sync-protocol-types   = { workspace = true }
+futures                        = { workspace = true }
+jsonrpsee                      = { workspace = true, features = ["macros", "server", "tracing"] }
+macros                         = { workspace = true }
+mpt-trusted-light-client-types = { workspace = true, features = ["serde"] }
+serde                          = { workspace = true, features = ["derive"] }
+tokio                          = { workspace = true }
+tracing                        = { workspace = true }
+unionlabs                      = { workspace = true }
+voyager-message                = { workspace = true }
+voyager-vm                     = { workspace = true }
```

Comment:
for a follow-up to do soon: run cargo udeps/ cargo machete on all Voyager modules & plugins